moderation/message/cog.py

```diff
@@ -178,6 +178,8 @@ async def edit_embed(self, ctx: Context, message: Message, color: Optional[Color
 
         if color is not None:
             send_embed.colour = color
+        elif color is None and message.embeds[0].color is not Embed.Empty:
+            send_embed.colour = int(("0x" + str(message.embeds[0].color).split("#")[1]), 16)
```

Comment:
Why do you make this soooo complicated? Fancy code does not make good code!

```suggestion
            send_embed.colour = message.embeds[0].color
```